pkg/cmd/server/origin/run_components.go

```diff
@@ -428,19 +429,24 @@ func (c *MasterConfig) RunImageImportController() {
 		limiter = flowcontrol.NewTokenBucketRateLimiter(importRate, importBurst)
 	}
 
-	factory := imagecontroller.ImportControllerFactory{
-		Client:               osclient,
-		ResyncInterval:       10 * time.Minute,
-		MinimumCheckInterval: time.Duration(c.Options.ImagePolicyConfig.ScheduledImageImportMinimumIntervalSeconds) * time.Second,
-		ImportRateLimiter:    limiter,
-		ScheduleEnabled:      !c.Options.ImagePolicyConfig.DisableScheduledImport,
+	ctrl, sched := imagecontroller.NewImageStreamControllers(
+		isInformer,
+		osclient,
+		time.Duration(c.Options.ImagePolicyConfig.ScheduledImageImportMinimumIntervalSeconds)*time.Second,
+		limiter,
+		!c.Options.ImagePolicyConfig.DisableScheduledImport,
+	)
+	if c.Options.ImagePolicyConfig.DisableScheduledImport {
+		glog.V(2).Infof("Scheduled image import is disabled - the 'scheduled' flag on image streams will be ignored")
```

Comment:
The image stream controller could know that and could add an annotation with that info. I'll defer to @smarterclayton for opinion, since he has mixed feelings, but I agree it would be beneficial for users to know this.